make

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+###################################################
+#  Miking is licensed under the MIT license.
+#  Copyright (C) David Broman. See file LICENSE.txt
+#
+#  To make the build system platform independent,
+#  building is done using Dune and called from
+#  this bash script (on UNIX platforms) or
+#  using make.bat (on Windows).
+###################################################
+
+
+
+# Install the server
+install() {
+    bin_path=$HOME/.local/bin/
+    mkdir -p $bin_path
+    cp -f build/mi $bin_path/mi; chmod +x $bin_path/mi
+    rm -rf $lib_path; cp -rf stdlib $lib_path
+}
```

Comment:
This doesn't seem to belong here.

Reply:
Might have forgotten it there. Deleted and pushed again